File: UI/ClientApp/src/app/Model/Subscription.ts
export enum SubscriptionType {
  Free,
  Timespan,
  Lifetime
}

export class Subscription {
  public trainerId: string = '';
  public athleteId: string = '';
  public subscriptionType: SubscriptionType = SubscriptionType.Free;
  public expiration: Date | undefined;
}

File: Core/Database/Entities/Subscription.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Lift.Buddy.Core.Database.Entities
{
    public enum SubscriptionType
    {
        Free,
        Timespan,
        Lifetime
    }
    public class Subscription
    {
        public Guid AthleteId { get; set; }
        public Guid TrainerId { get; set; }
        public SubscriptionType SubscriptionType { get; set; }
        public DateTime? Expiration { get; set; }

        public User Athlete { get; set; }
        public User Trainer { get; set; }

    }
}
